examples/md-flexible/src/domainDecomposition/RegularGridDecomposition.h

```diff
@@ -81,6 +81,12 @@ class RegularGridDecomposition final : public DomainDecomposition {
    */
   const std::array<int, 3> getDecomposition() { return _decomposition; }
 
+  /**
+   * Returns the number of subdomains in the simulation.
+   * @return number of subdomains
+   */
+  const int getNumberOfSubdomains();
```

Comment:
```suggestion
  int getNumberOfSubdomains() const;
```
`const` return type does not really make sense but the function itself is `const`. Please also check the other functions in this class for this.